src/layout.cpp

```diff
@@ -503,7 +503,7 @@ class LayoutParser
           QCString()
         }
       };
-      LayoutNavEntry::Kind kind;
+      LayoutNavEntry::Kind kind = static_cast<LayoutNavEntry::Kind>(0);
```

Comment:
this is not an improvement in my opinion. I think it is better to keep the enum uninitialized and then add a warning plus early return with an error on line 518 if line 514 was never executed.

Reply:
This is a bit a complex situation.
- the warning is already present from the block starting in line 519
- in line 499 we see `static_cast<LayoutNavEntry::Kind>(0),` 
- in line 506 the variable was not initialized, so what would have been the best guess? I chose 0 as above in line 499 (actually any value would do).
- we could introduce a `LayoutNavEntry::Unknown` but this might cause problems / warnings in other places (like defaults in switch statements). I'm not afraid that `LayoutNavEntry::Unknown`` would be used as this would have been caught by the test in line 519.

Reply:
Looks like I found a relatively easy solution.

Reply:
👍